src/annotator/highlight-clusters.tsx

```diff
@@ -63,6 +73,7 @@ export class HighlightClusterController implements Destroyable {
   private _features: IFeatureFlags;
   private _outerContainer: HTMLElement;
   private _shadowRoot: ShadowRoot;
+  private _updateTimeout?: number;
```

Comment:
The timeout needs to be cleared when the `HighlightClusterController` is torn down.

Reply:
Well spotted, thanks.